CRM/Upgrade/Incremental/sql/5.65.alpha1.mysql.tpl

```diff
@@ -1 +1,31 @@
 {* file to handle db changes in 5.65.alpha1 during upgrade *}
+
+-- Ensure new name field is not null/unique. Setting to ID is a bit lazy - but it works.
+UPDATE civicrm_group SET `name` = `id` WHERE name IS NULL;
+
+-- Add name field, make frontend_title required (in conjunction with php function)
+{if $multilingual}
+    {foreach from=$locales item=locale}
+      UPDATE `civicrm_group`
+      SET `frontend_title_{$locale}` = `title_{$locale}`
+      WHERE `frontend_title_{$locale}` IS NULL OR `frontend_title_{$locale}` = '';
+
+      UPDATE `civicrm_group`
+      SET `frontend_description_{$locale}` = `description`
+      WHERE `frontend_description_{$locale}` IS NULL OR `frontend_description_{$locale}` = '' AND `description` <> '';
```

Comment:
My eye just caught this now too. I think it's intended to be `(frontend_description_{$locale} IS NULL OR frontend_description_{$locale} = '') AND description <> ''` but it gets evaluated as `frontend_description_{$locale} IS NULL OR (frontend_description_{$locale} = '' AND description <> '')`. In mysql, unlike everything else, AND has a higher precedence than OR: https://dev.mysql.com/doc/refman/8.0/en/operator-precedence.html

Reply:
Looks like you fixed this too - thanks!